lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -35,6 +35,7 @@ const PAGE_HUNG_EXIT_CODE = 68;
 const INSECURE_DOCUMENT_REQUEST_EXIT_CODE = 69;
 const RETRIES = 3;
 const NUMERICAL_EXPECTATION_REGEXP = /^(<=?|>=?)((\d|\.)+)$/;
+const VERBOSE = process.env.LH_SMOKE_VERBOSE === '1';
```

Comment:
WDYT about just `Boolean(process.env.LH_SMOKE_VERBOSE)`?

Reply:
later down we use a env var `SMOKEHOUSE_DEBUG` maybe we should unify these two 🤔 

maybe we rename that one to `LH_SMOKE_DEBUG`?